src/plugins/plugin.filler.js

```diff
@@ -190,29 +190,66 @@ function isDrawable(point) {
 	return point && !point.skip;
 }
 
-function drawArea(ctx, curve0, curve1, len0, len1) {
-	var i;
+function drawArea(ctx, curve0, curve1, len0, len1, mode, area) {
+	var i, above, below;
 
 	if (!len0 || !len1) {
 		return;
 	}
 
+	switch (mode) {
+	case 'above':
+		above = true;
+		break;
+	case 'below':
+		below = true;
+		break;
+	default:
+		above = false;
+		below = false;
+	}
+	if (below || above) {
+		var clipPath = new Path2D();
```

Comment:
It seems this is not supported in IE or Safari.
https://developer.mozilla.org/en-US/docs/Web/API/Path2D#Browser_compatibility

Reply:
Thanks ! There was some other issues so I updated the entire patch. IE is now supported.